lib/linguist/languages.yml

```diff
@@ -64,8 +64,8 @@ ABAP CDS:
   type: programming
   color: "#555e25"
   extensions:
-    - ".asddls"
-  tm_scope: "source.abapcds"
```

Comment:
Not sure why this got changed. Think the update-ids script reindented?

Reply:
> Think the update-ids script reindented?

Yes. When it updates `languages.yml`, the entire file is replaced _in situ_ with the YAML-encoded representation of the in-memory database.

Here's what `update-ids` would look like if it were Node.js/JSON instead of Ruby/YAML:

~~~js
#!/usr/bin/env node
import {readFileSync, writeFileSync} from "fs";

const jsonFile = "./lib/linguist/languages.json";
let languages;

// Load database
languages = readFileSync(jsonFile, "utf8");
languages = JSON.parse(languages);

// Generate IDs for any entries that lack one
for(const name in languages){
	if(!("language_id" in languages[name]))
		languages[name].language_id = generateRandomID();
}

// Write updated database back to file
languages = JSON.stringify(languages, null, "\t");
writeFileSync(jsonFile, languages);
~~~

Reply:
So when you updated the IDs, the script naturally reformatted anything that was added manually in the time since `update-ids` was last run.

Reply:
Ok! should i revert or keep it in the PR?

Reply:
Keep it in the PR. This happens from time-to-time.